labellab_mobile/lib/screen/login/login_screen.dart

```diff
@@ -230,9 +230,9 @@ class _LoginScreenState extends State<LoginScreen> {
     Application.router.navigateTo(context, "/signup");
   }
 
-  void _showAuthFailSnackbar(BuildContext context) {
+  void _showAuthFailSnackbar(BuildContext context, String message) {
```

Comment:
@amlannandy thanks for the PR. It would be better if you could assign a default value for `_showAuthFailSnackbar`  function so you don't have to pass `null` every time you call it without a message. everything else looks pretty good 😀.

Reply:
@UdeshUK Thanks for the review! I'll make the necessary changes soon! 😃